lints/lint_issuer_multiple_rdn.go

```diff
@@ -0,0 +1,45 @@
+package lints
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
```

Comment:
Just let this be an empty struct, no comment needed.

`type IssuerRDNHasMultipleAttribute struct {}`